README.md

```diff
@@ -114,21 +114,6 @@ In [this](./src/isar/config/predefined_missions) folder there are predefined def
 corresponding to `mission_id=1`. A new mission may be added by adding a new json-file with a mission description. Note,
 the mission IDs must be unique.
 
-### Running with docker-compose
-
-ISAR may be started with an instance of the [isar-robot](https://github.com/equinor/isar-robot) package by
```

Comment:
Worth adding a reference to run isar-robot with docker? Or send them to the isar-robot repo to run it?

Reply:
I believe there is already a reference to isar-robot